src/controllers/imageController.js

```diff
@@ -0,0 +1,42 @@
+import {Images} from '../database'
+import { calculateLimitAndOffset, paginate } from "paginate-info";
+
+
+class imagesController {
+  static async search(req, res) {
+    try {
+      const { image } = req.params;
+      const { currentPage = 1, pageSize = 10 } = req.query;
+      const { limit, offset } = calculateLimitAndOffset(currentPage, pageSize);
+      const images = await Images.where("categories", "==", image).get();
```

Comment:
There is an error here on search

That is the error, I putted anything in postman path
<img width="1102" alt="Screen Shot 2021-06-10 at 22 22 35" src="https://user-images.githubusercontent.com/56052588/121593088-cdd37280-ca3b-11eb-823c-7f175b98e628.png">